src/sqlfluff/core/default_config.cfg

```diff
@@ -64,6 +64,7 @@ indented_ctes = False
 indented_using_on = True
 indented_on_contents = True
 indented_then = True
+indented_then_contents = True
```

Comment:
Let's not include this in the default config for now

Reply:
Why not? This just recovers the current behavior. If I'm not mistaken, the formatting for existing queries is changed if this is not included?

Reply:
Oh yeah - you're right!